config/routes.rb

```diff
@@ -50,6 +50,8 @@
 
     get '/interstitial', to: 'after_sign_in#interstitial', as: :interstitial
 
+    get '/:candidate_id/application/:id/submitted' => 'application_form#review_previous_application', as: :review_previous_application
```

Comment:
Why do we need this?

Reply:
Theo asked for the work to be completed that would allow a candidate to see all of their previous applications. So moving it over from using the previous_application to taking an application_form id seemed like the most reasonable thing to do

Reply:
Sorry, I meant `:candidate_id`. It's not used?

Reply:
Oh right. Yea good point 👍